scss/civicrm/administer/settings/_extensions.scss

```diff
@@ -1,7 +1,12 @@
 &.page-civicrm-admin-extensions {
+  .help {
+    margin: 20px;
```

Comment:
why does the `.help` component has this margin bottom only for this page? why the exception

Reply:
This is because this page is having .help section inside elements while other pages have it outside. so, we are giving margin from surrounding to make it look proper 
![image](https://user-images.githubusercontent.com/26058635/35218546-107940f8-ff95-11e7-8d64-fbae09aa5c99.png)